### ISSUE TITLE
Missing field in the schema description (researcher/department.name)

### ISSUE THREAD

**Issue by pkiraly** [LibreCat/LibreCat#41]
From the automatically generated schema Wiki page the researcher's department name is missing.

Here is an extract from the devel/researcher.yml, which shows the missing field:

```
department:
- _id: 9996
  name: 'Relativistic Theory Group'
```

In the Wiki page only the department._id is declared, the department.name is missing.

**Comment by phochste** [comment]
I didn't add this deliberately not in the schema. I would find it quite strange to have a requirement for including both department._id and department.name as required parameters when creating an researcher. If you know the department._id it would be enough.

@vpeil It is true that these department.name in researcher records have any use right now? And if it has a use..why?

**Comment by vpeil** [comment]
@phochste correct. We should store `department._id` only. The rest can be looked up from the department bag. I'll change the example file.

**Comment by vpeil** [comment]
fixed in 2157e62eb40d454bee000e4c5d59dfe3c53e0e72
